content/professionalism/business-selfie/_index.md

```diff
@@ -5,8 +5,10 @@ flavours:
 - none
 submission_type: link
 title: Business selfie
+ready: true
 ---
 
+
```

Comment:
Can you see that it says `DEPRECATED`? We would not want to show this content to our learners like this. This means it is not ready. So ready should be false.

If something is ready it means it is ready for our learners to use. This is not ready. We need the configuration to tell the truth.